test/e2e/specs/site-editor/preload.spec.js

```diff
@@ -44,11 +44,11 @@ test.describe( 'Preload', () => {
 
 		// To do: these should all be removed or preloaded.
 		expect( requests ).toEqual( [
+			// Seems to be coming from `enableComplementaryArea`.
+			'/wp/v2/users/me',
```

Comment:
Had to change the order here. Now that `getTemplateId` uses the REST index, the settings request is triggered a bit later.